share/instruments/instruments.xml

```diff
@@ -10429,7 +10429,7 @@
             </Instrument>
       </InstrumentGroup>
       <InstrumentGroup id="strings">
-            <name>Strings</name>
```

Comment:
I like the less english but more structured way. It will make sorting more visual.